civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -74,35 +74,8 @@
 
 /*
 * TEMP fixes till ui-select styling is fixed
+* Dropdown ul opacity was set to 0
 */
-.civihr-ui-select {
```

Comment:
related to the comment i made in the other pr, why are we removing the entire civihr-ui-select theme? If we had it before, it means it was necessary. What happened that we can just discard it completely? You give no details in the PR description